adding_metadata_sql.md

```diff
@@ -0,0 +1,181 @@
+## SQL
+
+SQL can also be used for bulk uploads. However, SQL should primarily be used in the following cases:
+
+* fix data errors 
+* remove duplicate records
```

Comment:
Don't use "record" to refer to an SQL database row.  "Record" should only be used in the colloquial sense as in "We have a record of the sorghum yield for 2016 stored in a row of the yields table in the BETYdb database." or in the technical sense used in the PostgreSQL documentation (see the table at https://www.postgresql.org/docs/9.4/datatype-pseudo.html or https://www.postgresql.org/docs/9.4/plpgsql-declarations.html#PLPGSQL-DECLARATION-RECORDS).  

([Some database people feel especially strongly about this.](https://i.stack.imgur.com/oIoG7.jpg))

But more to the point, this list is redundant and vague: The second and third item are subsumed under the first, but "fix data errors" is too broad.  There are all sorts of data errors that can be fixed just as easily and more safely by using the BETYdb Rails app.  The opening statements of this section might better read "SQL should be used only when the data manipulation to be done either can't be done using the web interface or BETYdb-YABA or where doing so would be exceedingly tedious."  Then give examples, which might include
1. correcting systemic data errors involving large numbers of rows
2. assigning non-point geometries to sites [this might be possible in YABA, I don't know]
3. associating experiments with sites and treatments
4. associating sites with cultivars

It's probably worth noting which of the steps below can't be done in the web interface (6, 7, and 9, and assigning a non-point geometry in 2) or the YABA interface.